src/mgr/MetricTypes.h

```diff
@@ -229,6 +238,14 @@ struct MetricConfigMessage {
     : payload(payload) {
   }
 
+  bool should_encode(uint64_t features) const {
+    if (!HAVE_FEATURE(features, SERVER_PACIFIC) &&
+	boost::get<MDSConfigPayload>(&payload)) {
+      return false;
+    }
+    return true;
```

Comment:
```suggestion
    return (HAVE_FEATURE(features, SERVER_PACIFIC) ||
	      boost::get<MDSConfigPayload>(&payload));
```

Reply:
I think that condition would be backwards: if it's an octopus peer and we store and MDS payload, we *don't* want to encode it.